tests/tests/fixtures/querygen/mod.rs

```diff
@@ -159,3 +159,106 @@ where
 
     Ok(())
 }
+
+/// Compare with setup SQL for repro script generation
```

Comment:
This is an improvement to our proptest to print the repro at the end of failure.